kuzzleio/documentation
# SPA mode

## What does this PR do?
Replaces all the `<a>` tags in the `Sidebar` component (and others) with `<router-link>` so that the entire website behaves like an SPA after first load.

Thanks to @alexandrebouthinon for helping on this! :tada: 

### How should this be manually tested?
Build the docs locally and navigate with the Network tab of your Devtools open. You'll see that only a tiny js script is loaded after clicking on a link.

### Other changes
The same substitution is operated on the following components

* Redirect.vue
* SDKIndex.vue
* Tabs.vue
* TabsMobile.vue
